### PR TITLE
Update opam files and cleanup travis

### PR DESCRIPTION
**Pull request by mseri**
Signed-off-by: Marcello Seri <marcello.seri@gmail.com>

### CODE REVIEW

**Review by rlepigre — APPROVED** [review]

**Comment by rlepigre** [comment]
Feel free to merge or push on master directly, you seem to know what you are doing more than I do (and you should have the right now). In any case, thanks!

File: imagelib-unix.opam [hunk 1]
```diff
@@ -23,13 +23,13 @@ authors: [
   "Rodolphe Lepigre <rodolphe.lepigre@inria.fr>"
 ]
 license: "GPLv3"
-doc: "https://github.com/rlepigre/imagelib"
+doc: "https://rlepigre.github.io/imagelib"
```
Comment (hannesm) [inline imagelib-unix.opam]:
```suggestion
doc: "https://rlepigre.github.io/ocaml-imagelib"
```

File: imagelib.opam [hunk 2]
```diff
@@ -29,11 +29,11 @@ authors: [
   "Rodolphe Lepigre <rodolphe.lepigre@inria.fr>"
 ]
 license: "GPLv3"
-doc: "https://github.com/rlepigre/imagelib"
+doc: "https://rlepigre.github.io/imagelib"
```
Comment (hannesm) [inline imagelib.opam]:
```suggestion
doc: "https://rlepigre.github.io/ocaml-imagelib"
```

**Comment by mseri** [comment]
Thanks @hannesm, well spotted

**Comment by mseri** [comment]
With the latest changes also travis should work, so that we have a proper run of tests before releasing

**Comment by mseri** [comment]
I don't understand why opam in Travis fails to detect `imagelib`. Locally here it seems to work, but I fear that it is because the fylesystem of my laptop is case insensitive... Can someone with a linux install try?

**Comment by mseri** [comment]
I am going to merge this, since it will be useful for the release, but I will not release until we understand why the Travis build is failing. I hope to have access to a linux box soon

**Pull request merged**